### ISSUE TITLE
How to wait for all the responses to finish?

### ISSUE THREAD

**Issue by isi-gach** [Netflix/pollyjs#71]
Is there anything like `fetchMock.fush()` ?
http://www.wheresrhys.co.uk/fetch-mock/api#flush
This is how they do it:
```
FetchMock.flush = function() {
	return Promise.all(this._holdingPromises);
};
```
```
FetchMock.fetchHandler = function(url, opts) {
	const response = this.executeRouter(url, opts);

	[...]

	// this is used to power the .flush() method
	let done;
	this._holdingPromises.push(new this.config.Promise(res => (done = res)));

	// wrapped in this promise to make sure we respect custom Promise
	// constructors defined by the user
	return new this.config.Promise((res, rej) => {
		this.generateResponse(response, url, opts)
			.then(res, rej)
			.then(done, done);
	});
};
```

**Comment by jasonmit** [comment]
> Is there anything like `fetchMock.fush()`?

There isn't.

To better understand what our API could possibly be, I'm curious how you planned to use it.  Can you whip up a pseudo test?

**Comment by offirgolan** [comment]
@isi-gach a fix has been published with `v1.1.0` of `@pollyjs/core`. You will also have to upgrade your adapters to their latest versions.
